lib/systemservice/systemd.go

```diff
@@ -332,19 +345,19 @@ func (s *systemdManager) UninstallService(req UninstallServiceRequest) error {
 	serviceName := serviceName(req.Name)
 	out, err := invokeSystemctl("stop", serviceName)
 	if err != nil && !IsUnknownServiceError(err) {
-		return trace.Wrap(err, out)
+		return trace.Wrap(err)
```

Comment:
I'm not sure it's better to drop the out string from systemctl? Specifically on error does this not provided more context?

Reply:
Oh, I did not want that - my bad.